src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+
+    private static final double FORMULA_NUMBER = 2;
```

Comment:
It is stores double literal?

Reply:
Just make this constant int, cause you store int value there